addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -268,6 +268,24 @@
         }
     }
 
+    %o_we_sublevel > we-title::before {
```

Comment:
I had to move this part in order to be able to use `o_we_sublevel` inside the "Invisible Elements" panel (`.o_we_invisible_el_panel`)